apps/camera/js/controllers/preview-gallery.js

```diff
@@ -77,6 +77,11 @@ PreviewGalleryController.prototype.openPreview = function() {
   this.view.on('loadingvideo', this.app.firer('busy'));
   this.view.on('playingvideo', this.app.firer('ready'));
 
+  // Screen reader right/left swipes with two fingers to navigate to
+  // next/previous frames.
+  this.view.on('wheel:right', this.next);
+  this.view.on('wheel:left', this.previous);
```

Comment:
@justindarc , yes, screen reader synthesizes the wheel event when the screen reader user performs a swipe with 2 fingers.